### PR TITLE
add more usage for github gist

### PR DESCRIPTION
**Pull request by daleione**

### CODE REVIEW

**Comment by robbyoconnor** [comment]
NO NO NO NO NO, add leader ksys for this functionality :)

File: layers/+source-control/github/README.org [hunk 1]
```diff
@@ -79,6 +79,19 @@ In the gist list buffer:
...
+|-------------+--------------------------------|
+| C-x C-s     | save a new version of the gist |
+| C-x C-w     | rename some file               |
+
+Dired integration. From a dired buffer, you can:
+
+| Key Binding | Description                                                      |
+|-------------+------------------------------------------------------------------|
+| @           | make a gist out of marked files (with a prefix, make it private) |
+
```
Comment (robbyoconnor) [inline layers/+source-control/github/README.org]: Can you add some spacemacs keybindings? I don't use emacs editing style

**Comment by robbyoconnor** [comment]
Add spacemacs leader key keybindings -- I don't think this should be merged as-is.

**Comment by StreakyCobra** [comment]
@robbyoconnor Proposing a set of keys and pointers to existing keybindings definition code would be more helpful probably ;-)

**Comment by robbyoconnor** [comment]
@StreakyCobra -- probably - I don't feel like coming up with them -- nor do I care -- I just don't think this should be merged as-is -- unless the namespace doesn't exist (happened for my emberjs layer) -- then it's fine -- but it exists

**Comment by nixmaniack** [comment]
I think there's no added functionality in this PR. Functionality is already there, just documentation needs right keybindings stated. `SPC f s` for `save new version` and `SPC f R` for `rename` ? I haven't tested this, just pointing out.

**Comment by robbyoconnor** [comment]
Why? Just rebase...

**Comment by nixmaniack** [comment]
Docs aren't updated. Let's keep open!

**Comment by smile13241324** [comment]
Sorry guys but this PR is now over 4 years old, we will not be able to merge this into the current code base anymore therefore I will close this. If we still need this, a new PR should be opened following the current conventions and bases to develop.

**Pull request closed without merging**